tests/unit/test_cache_factory.py

```diff
@@ -19,6 +21,18 @@ def in_memory_cache_config():
     )
 
 
+@pytest.fixture(scope="module")
+def redis_cache_config():
+    """Fixture containing initialized instance of ConversationCacheConfig."""
+    # os.environ["OLS_CONVERSATION_CACHE"] = constants.REDIS_CACHE
```

Comment:
Why is the commented line here?

Reply:
is shows, how it can be configured on envvar level, see `InMemoryCache` unit tests

Reply:
I see it there too, but why is this particular information in tests? Especially without further description. It just looks like forgotten code.

Reply:
ok removed @onmete